Packs/DigitalShadows/Integrations/ReliaquestTakedown/ReliaquestTakedown.py

```diff
@@ -0,0 +1,600 @@
+    json_data = r.json()
+    return json_data
+
+
+def list_brands(request_handler: Client, args, **kwargs) -> list:
+    """
+    Return takedown brands for given customer
+    Arguments:
+      request_handler (HttpRequestHandler): the request handler to use to make HTTP requests
+      args: arguments sent in the command as input
+    """
+    url = "/v1/takedown-brands"
+    r = request_handler.get(url, params={}, **kwargs)
+    r.raise_for_status()
+    json_data = r.json()
+    return json_data
+
+
+def create_comment(request_handler: Client, args, **kwargs) -> list:
+    """
+    Crate takedown comment for given takedown id
+    Arguments:
+      request_handler (HttpRequestHandler): the request handler to use to make HTTP requests
+      args: arguments sent in the command as input
+    """
+    takedown_id = args.get("takedown_id")
+    comment = args.get("comment")
+    payload = {"content": comment}
+    url = f"/v1/takedowns/{takedown_id}/comments"
+    demisto.debug(f"creating takedown comment with payload: {payload}")
+    r = request_handler.post(url, data=payload)
+    r.raise_for_status()
+    json_data = r.json()
+    return json_data
+
+
+def upload_attachment(request_handler: Client, args, **kwargs) -> list:
+    """
+    Upload file as a attachment to the takedown
+    Arguments:
+      request_handler (HttpRequestHandler): the request handler to use to make HTTP requests
+      args: arguments sent in the command as input
+    """
+
+    file_id = demisto.args().get("file_id")
+    takedown_id = args.get("takedown_id")
+
+    result = demisto.getFilePath(file_id)
+
+    with open(result["path"], "rb") as file:
+        # Prepare the files dictionary for the POST request
+        files = {
+            "file": (result["name"], file, "application/octet-stream")  # (field_name, file_object, mime_type)
+        }
+        url = f"/v1/takedowns/{takedown_id}/attachments"
+
+        response = request_handler.post(url, files=files, data={})
+        if response.status_code == 200:
+            return response.json()
+        else:
+            return response.text
+
+
+def download_attachment(request_handler: Client, args, **kwargs) -> list:
+    """
+    Download file as a attachment from the takedown
+    Arguments:
+      request_handler (HttpRequestHandler): the request handler to use to make HTTP requests
+      args: arguments sent in the command as input
+    """
+    attachment_id = args.get("attachment_id")
+
+    url = f"/v1/takedowns/attachments/{attachment_id}/download"
+    demisto.debug(f"downloading file for attachment: {attachment_id}")
+    r = request_handler.get(url, data={})
+    r.raise_for_status()
+
+    path = r.headers.get("Content-Disposition").split(";")[-1].split("=")[-1].replace('"', "")
+    with open(path, "wb") as file:
+        for chunk in r.iter_content(chunk_size=8192):  # Download in chunks
+            file.write(chunk)
+
+    file_entry = fileResult(path, open(path, "rb").read())
```

Comment:
```suggestion
    file_entry = fileResult(path, r.content)
```
As we discussed in our call.